src/cli.lisp

```diff
@@ -44,8 +49,14 @@
        ,@body)))
 
 
-;;; Qi Install ($ qi --install [package] / $ qi -i [package]) internals
+;;; Qi install-deps ($ qi --install-deps project.asd)
+(defun opt-install-deps (input)
+  "Install the dependencies locally for the system definition file provided as INPUT."
+  (let* ((proj (pathname-name input)))
```

Comment:
`let` isn't even necessary anymore, let alone `let*`; that's what I get for doing this at 6AM.